dnf/dnf-utils.c

```diff
@@ -36,12 +70,76 @@ dnf_utils_print_transaction (DnfContext *ctx)
       g_print ("Nothing to do.\n");
       return FALSE;
     }
-  g_print ("Transaction: %u packages\n", pkgs->len);
 
-  for (guint i = 0; i < pkgs->len; i++)
+  struct libscols_table *tb;
+  struct libscols_line *ln;
+
+  tb = scols_new_table ();
+  scols_table_new_column (tb, " Package",   0.7, SCOLS_FL_TRUNC);
```

Comment:
The leading space in " Package" is not nice, we'll have to change it once microdnf supports translations anyway.